cluster-autoscaler/metrics/metrics.go

```diff
@@ -263,10 +264,11 @@ func UpdateClusterSafeToAutoscale(safe bool) {
 }
 
 // UpdateNodesCount records the number of nodes in cluster
-func UpdateNodesCount(ready, unready, starting int) {
+func UpdateNodesCount(ready, unready, starting, longUnregistered int) {
```

Comment:
Maybe it would be useful to also add unregistered nodes (not longUnregistered)? Seems we could easily get the number by adding a method to CSR that would return `len(unregisteredNodes) - totalReadiness.LongUnregistered`. Though I only had a quick glance at the code, so may be missing something.

Reply:
Done. Added unregistered to Readiness struct.